package.json

```diff
@@ -1,25 +1,24 @@
 {
   "name": "nomidot",
-  "version": "1.0.0",
-  "main": "index.js",
-  "author": "",
+  "author": "Parity Technologies <admin@parity.io>",
+  "description": "Staking Portal for Polkadot",
   "license": "Apache-2.0",
+  "private": true,
+  "repository":"https://github.com/yjkimjunior/Nomidot",
+  "scripts": {
+    "prelerna:publish": "lerna exec yarn build --stream --scope @substrate/context --scope @substrate/ui-components",
```

Comment:
small thing but could we just make this `lerna:pre` and `lerna:publish`?

Reply:
hmm, it's like a package.json convention, if you a script `{s}` and another `pre{s}`, then `yarn {s}` will run `yarn pre{s}` first

Reply:
It's basically to save us from running 2 scripts to publish

Reply:
I could also put everything into one long script, if that's clearer to you

Reply:
ah okay, i didn't know that, let's jsut leave it as is in that case